src/radical/pilot/agent/scheduler/continuous_reconfig.py

```diff
@@ -0,0 +1,57 @@
+
+__copyright__ = 'Copyright 2023, The RADICAL-Cybertools Team'
+__license__   = 'MIT'
+
+import os
+
+import radical.utils as ru
+
+from ... import constants as rpc
+
+from .continuous import Continuous
+
+
+# ------------------------------------------------------------------------------
+#
+class ContinuousReconfig(Continuous):
+
+    # --------------------------------------------------------------------------
+    #
+    def __init__(self, cfg, session):
+
+        super().__init__(cfg, session)
+
+        self._task_reqs = {}  # resource requirements (ranks, cores_per_rank)
+
+    # --------------------------------------------------------------------------
+    #
+    def _set_task_reqs(self):
+
+        reqs_file = ''
```

Comment:
provide file name through the pilot environment (set within the PilotDescription) @andre-merzky

Reply:
ping @andre-merzky